### ISSUE TITLE
crash report

### ISSUE THREAD

**Issue by toufikbelouadah** [ranger/ranger#1252]
- Operating system and version:  Linux blackpearl 4.17.12-arch1-1-ARCH #1 SMP PREEMPT Fri Aug 3 07:16:41 UTC 2018 x86_64 GNU/Linux

 Terminal emulator and version: rxvt-unicode (urxvt) v9.22 - released: 2016-01-23
options: perl,xft,styles,combining,blink,iso14755,unicode3,encodings=eu+vn+jp+jp-ext+kr+zh+zh-ext,fade,transparent,tint,XIM,frills,selectionscrolling,wheel,slipwheel,cursorBlink,pointerBlank,scrollbars=plain+rxvt+NeXT+xterm

#### Traceback

```
ranger version: ranger-master 1.9.1
Python version: 3.7.0 (default, Jul 15 2018, 10:44:58) [GCC 8.1.1 20180531]
Locale: en_US.UTF-8
Current file: '/home/ghosty/Documents/the_psychonaut_field_manual_fourth_pdf_edition_by_bluefluke-d8rjuxc.pdf'

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/pager.py", line 236, in _generate_lines
    raise StopIteration
StopIteration

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/ranger/core/main.py", line 196, in main
    fm.loop()
  File "/usr/lib/python3.7/site-packages/ranger/core/fm.py", line 390, in loop
    ui.redraw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/ui.py", line 338, in redraw
    self.draw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/ui.py", line 365, in draw
    DisplayableContainer.draw(self)
  File "/usr/lib/python3.7/site-packages/ranger/gui/displayable.py", line 256, in draw
    displayable.draw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/view_miller.py", line 100, in draw
    DisplayableContainer.draw(self)
  File "/usr/lib/python3.7/site-packages/ranger/gui/displayable.py", line 256, in draw
    displayable.draw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/browsercolumn.py", line 176, in draw
    self._draw_file()
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/browsercolumn.py", line 203, in _draw_file
    Pager.draw(self)
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/pager.py", line 100, in draw
    for line, i in zip(line_gen, range(self.hei)):
RuntimeError: generator raised StopIteration

ranger crashed. Please report this traceback at:
https://github.com/ranger/ranger/issues

```

**Comment by Gomme-** [comment]
Looks like the same crash I have reported as well . See #1251

**Comment by toufikbelouadah** [comment]
weired beahviour it looks like ranger fails to handle the text preview of some pdf files

**Comment by toonn** [comment]
Was already fixed in #1244, this is because python made breaking changes in 3.7. Either wait for the next release or run the version on master (ranger-git in the AUR) or downgrade python.
